tests/newsletters/test_newsletter_views.py

```diff
@@ -12,12 +13,15 @@
 
 @pytest.mark.django_db
 def test_send_organisation(admin, client, project, user_factory,
-                           follow_factory):
+                           follow_factory, email_address_factory):
     organisation = project.organisation
     user1 = user_factory(get_newsletters=True)
-    user2 = user_factory()
+    user2 = user_factory(get_newsletters=True)
```

Comment:
This was to test if only one email is sent when one user does not want to get newsletters. In the new version it would make sense to have 3 users: one with email verified and get_newsletters, one with get_newsletters, but email not verified and one without get_newsletters, but email verified. Then there should still be one email sent.

Reply:
I thought this test was to test if only users that follow an organsisation receive the newsletter. That is why I added the get_newsletter=True, bc otherwise it would not have be sent anyway and thus the follow would have not been tested. 

The setting with the 3 users you describe above is tested in test_send_newsletter_platform(). There, user1 wants newsletters and is verified, user2 wants newsletters and is not verified and user3 doesnt want newsletters, but is verified. The email is then only sent to user1 (plus the initiator, moderator and admin, since they are all verified and want newsletters).

Reply:
Yeah, maybe the two different users are enough. Just thought the more different scenarious the better. :)